subjects/card_deck/README.md

```diff
@@ -2,46 +2,44 @@
 
 ### Instructions
 
-Represent cards from a desk
-
-- A standard deck of cards has 52 cards: 4 suits and 13 cards per suit
-
-- Start by creating the `Suit` enum and implement the associated
-function `random` which returns a random `Suit` (`Heart`,
-`Diamond`, `Spade` or `Club`)
+A standard deck of cards has 52 cards: 4 suits and 13 cards per suit.
+Represent the cards from a deck:
 
+- Start by creating the `Suit` enum
+- implement the associated **function** `random` which returns a random `Suit` (`Heart`, `Diamond`, `Spade` or `Club`)
 - Then create the `Rank` enum that can have the value
-`Ace`, `King`, `Queen`, `Jack`, and `Number` associated to an `u8`
-value to represent the ranks 2 through 10
-After create an associated function to `Rank` called `Random` that
-returns a random `Rank`
-
+  `Ace`, `King`, `Queen`, `Jack`, and `Number` associated to an `u8`
+  value to represent the ranks 2 through 10
+- After create an associated **function** to `Rank` called `Random` that
+  returns a random `Rank`
 - Finally create a structure name `Card` which has the fields `suit`
-and `rank`
+  and `rank`
 
 Define:
 
-The associated function `translate` for Rank and Suite
-- for `Suit`, `tranlate` makes the translation between an integer value (u8) and the suit of a card (1 -> Heart, 2 -> Diamonds, 3 -> Spade, 4 -> Club)
-- for `Rank`, `translate` makes the tranlation between an integer value (u8) and the rank ( 1 -> Ace, 2 -> 2, .., 10 -> 10, 11 -> Jack, 12 -> Queen, 13 -> King)
+- The associated **function** `transpose` for `Rank` and `Suit`:
+  - For `Suit`, `transpose_suit` makes the translation between an integer value(u8) and the suit of a card (1 -> Heart, 2 -> Diamonds, 3 -> Spade, 4 -> Club)
+  - For `Rank`, `transpose_rank` makes the translation between an integer value(u8) and the rank ( 1 -> Ace, 2 -> 2, .., 10 -> 10, 11 -> Jack, 12 -> Queen, 13 -> King)
+- The associated **function** `random` for `Rank` and `Suit` which returns a random `Rank` and `Suit` respectively
+- Finally define the **function** `winner_card` which returns `true` if the card passed as an argument is an Ace of spades
 
-The associated function `random` for `Rank` and `Suit` which returns a random rand and suit respectively
+### Notions
 
-Finally define the function `winner_card` that returns true if the card passed as an argument is an Ace of spades
+[Crate rand](https://docs.rs/rand/0.5.0/rand/)
 
 ### Expected Functions and Structures
 
 ```rust
 pub fn random() -> Suit {
 }
 
-pub fn translate(value: u8) -> Suit {
+pub fn transpose_suit(value: u8) -> Suit {
```

Comment:
Yes, there was a typo. Both should be name `translate`. There is no need to add the prefix _card and _deck because associated functions are how the name says associated to the type. So you will have to call them like this: Card::translate_card which is actually bad practice. Notice the useless repetition of the word card.